app/models/project.rb

```diff
@@ -1,4 +1,14 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def average_experience_of_contestants
+    total_experience = 0
+    contestants.each do |contestant|
+      total_experience += contestant.years_of_experience
+    end
+    (total_experience.to_f / contestants.count).round(2)
```

Comment:
While this logic is correct, this is a use of Ruby where AR would suffice. A "red flag" for you to remember going forward is the use of `.each` in a model method like you have here - if you are needing to iterate through a collection, chances are high that there's already an AR method to use instead. Ex: 

```
def average_experiecne_of_contestants
   contestants.average(:years_of_experience)
end
```
Try that out in pry and see what you get back!